depmapomics/constants.py

```diff
@@ -295,60 +293,114 @@
-    "dbsnp_id": "DbsnpID",
-    "dbsnp_filter": "DbsnpFilter",
-    "issues": "Issues",
-    "gc_content": "GcContent",
-    "lineage_association": "LineageAssociation",
-    "cancer_molecular_genetics": "CancerMolecularGenetics",
-    "ccle_deleterious": "CCLEDeleterious",
-    "structural_relation": "StructuralRelation",
-    "cosmic_hotspot": "CosmicHotspot",
-    "cosmic_overlapping_mutations": "CosmicOverlappingMutations",
-    "associated_with": "AssociatedWith",
-    "lof": "LoF",
-    "driver": "Driver",
-    "likely_driver": "LikelyDriver",
-    "transcript_likely_lof": "TranscriptLikelyLoF",
-    "civic_id": "CivicID",
-    "civic_description": "CivicDescription",
-    "civic_score": "CivicScore",
-    "popaf": "Popaf",
-    "likely_gof": "LikelyGoF",
-    "likely_lof": "LikelyLoF",
-    "hess_driver": "HessDriver",
-    "hess_signture": "HessSignature",
-    "hess_signature": "HessSignature",
-    "revel_score": "RevelScore",
-    "pharmgkb_id": "PharmgkbID",
-    "dida_id": "DidaID",
-    "dida_name": "DidaName",
-    "gwas_disease": "GwasDisease",
-    "gwas_pmid": "GwasPmID",
-    "gtex_gene": "GTexGene",
-    "DepMap_ID": "DepMap_ID",
+    'chrom': 'Chrom',
+    'pos': 'Pos',
+    'ref': 'Ref',
+    'alt': 'Alt',
+    'af': 'AF',
+    'dp': 'DP',
+    'ref_count': 'RefCount',
+    'alt_count': 'AltCount',
+    'gt': 'GT',
+    'ps': 'PS',
+    'variant_type': 'VariantType',
+    'variant_info': 'VariantInfo',
+    'dna_change': 'DNAChange',
+    'protein_change': 'ProteinChange',
+    'hugo_symbol': 'HugoSymbol',
+    'ensembl_gene_id': 'EnsemblGeneID',
+    'ensembl_feature_id': 'EnsemblFeatureID',
+    'hgnc_name': 'HgncName',
+    'hgnc_family': 'HgncFamily',
+    'uniprot_id': 'UniprotID',
+    'dbsnp_rs_id': 'DbsnpRsID',
+    'gc_content': 'GcContent',
+    'lof_gene_name': 'LofGeneName',
+    'lof_gene_id': 'LofGeneId',
+    'lof_number_of_transcripts_in_gene': 'LofNumberOfTranscriptsInGene',
+    'lof_percent_of_transcripts_affected': 'LofPercentOfTranscriptsAffected',
+    'nmd': 'NMD',
+    # 'clnsig': 'ClnSig',
+    'molecular_consequence': 'MolecularConsequence',
+    # 'af_exac': 'AfExac',
+    # 'af_esp': 'AfEsp',
+    # 'af_tgp': 'AfTgp',
+    'vep_impact': 'VepImpact',
+    'vep_biotype': 'VepBiotype',
+    'vep_hgnc_id': 'VepHgncID',
+    'vep_existing_variation': 'VepExistingVariation',
+    'vep_mane_select': 'VepManeSelect',
+    'vep_ensp': 'VepENSP',
+    'vep_swissprot': 'VepSwissprot',
+    'sift': 'Sift',
+    'polyphen': 'Polyphen',
+    'gnomade_af': 'GnomadeAF',
+    'gnomadg_af': 'GnomadgAF',
+    'vep_clin_sig': 'VepClinSig',
+    'vep_somatic': 'VepSomatic',
+    'vep_pli_gene_value': 'VepPliGeneValue',
+    'vep_loftool': 'VepLofTool',
+    'oncogene_high_impact': 'OncogeneHighImpact',
+    'tumor_suppressor_high_impact': 'TumorSuppressorHighImpact',
+    # 'achilles_top_genes': 'AchillesTopGenes',
+    # 'structural_relation': 'StructuralRelation',
+    # 'associated_with': 'AssociatedWith',
```

Comment:
@qinqian on a second thought I realize it might make more sense to remove structural_relation and associated_with because they were inferred from abandoned funcotator columns

Reply:
sure, that makes sense.